### PR TITLE
Support service dump

### PR DESCRIPTION
**Pull request by hzxuzhonghu**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: api/v2/workloadapi/workload_helper.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+package workloadapi
+
+// ResourceName returns the unique key of Workload.
+func (x *Workload) ResourceName() string {
+	return x.Uid
+}
+
+// ResourceName returns the unique key of Service.
+func (x *Service) ResourceName() string {
```
Comment (LiZhenCheng9527) [inline api/v2/workloadapi/workload_helper.go:25]: Would changing the name to GetResourceName be better?

Comment (hzxuzhonghu) [reply api/v2/workloadapi/workload_helper.go:25]: nit：I donot think so, it is actually generating resourceName.  `ResourceName` looks much better/simpler to me.

**Pull request merged**